build.sbt

```diff
@@ -32,15 +32,19 @@ inThisBuild(
 lazy val testSettings = List(
   testOptions.in(Test) +=
     Tests.Argument(TestFrameworks.ScalaCheck, "-verbosity", "2"),
+  testFrameworks := List(
+    new TestFramework("munit.Framework"),
+    new TestFramework("org.scalacheck.ScalaCheckFramework")
+  ),
   libraryDependencies ++= {
     if (SettingKey[Boolean]("nativeLinkStubs").?.value.contains(true))
       List(
-        "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % Test,
+        "org.scalameta" %%% "munit" % "0.4.5",
         "com.github.lolgab" %%% "scalacheck" % "1.14.1" % Test
       )
     else
       List(
-        "org.scalatest" %%% "scalatest" % "3.0.8" % Test,
+        "org.scalameta" %%% "munit" % "0.4.5",
```

Comment:
Was this meant to be in Compile scope?

Just noticed it's a transitive dependency of scalafix-core.

Reply:
Whoop, that explains why MUnit has so many downloads 🙈 😓 😰 The Scalafix download numbers are only ~1.4k for scalafix-core_2.12:0.9.12+ but I see MUnit is also a transitive dependency of scalafmt-core, which has a lot more downloads 😅

Reply:
Fixed in 	https://github.com/olafurpg/metaconfig/pull/86 I'll also correct my tweet.

Reply:
It's maybe not so bad, the latest scalafmt versions only have ~5k unique IP downloads. The most downloaded versions don't depend on munit

<img width="945" alt="Screenshot 2020-04-10 at 10 22 03" src="https://user-images.githubusercontent.com/1408093/78984192-2dbb4a00-7b15-11ea-8c2b-ea0ec452b39d.png">
<img width="962" alt="Screenshot 2020-04-10 at 10 15 31" src="https://user-images.githubusercontent.com/1408093/78984194-2eec7700-7b15-11ea-891b-41ea2d4234a9.png">